### PR TITLE
PlayerView post-combat stats

### PR DESCRIPTION
**Pull request by coreyjkelly**
More re: #423 

Curious what you think of this implementation. The react component is completely re-computed, but that might be necessary if we want the stats to be optional in each view.

Some of the design at the `PlayerView.tsx` and `ReactPlayerView.tsx` is a bit clunky, but I think without a full generalized way to deal with "Prompts" at that level, this might be fine. I can't think of many other use cases for pushing a prompt to `PlayerView` that would require it.

Also, I'm no good at CSS, so I've exactly copied the `EncounterView` style. Looks alright in a browser, but probably not great on mobile. Feedback there would be appreciated.

### CODE REVIEW

File: client/Player/components/PlayerView.tsx [hunk 1]
```diff
@@ -25,6 +27,7 @@ interface LocalState {
 interface OwnProps {
   onSuggestDamage: ApplyDamageCallback;
   onSuggestTag: (combatantId: string, tagState: TagState) => void;
+  combatStats: CombatStats;
```
Comment (cynicaloptimist) [inline client/Player/components/PlayerView.tsx]: You could make this property optional here:
`combatStats?: CombatStats;`
This allows it to be `undefined`, so you could omit most of the lines in tests, etc. where you're setting it to `null`.

**Comment by cynicaloptimist** [comment]
Some CSS issues in Chrome for edge cases like this:
image
Functionality and code looks good. I'd like clicking anywhere outside of the modal to dismiss the prompt, then you could omit the `<SubmitButton />` which does the wrong thing anyways if used outside of a prompt.

**Comment by coreyjkelly** [comment]
Could probably still use some input from a CSS person, but I think the main issues are worked out.

File: lesscss/player-view.less [hunk 2]
```diff
@@ -113,6 +113,70 @@
...
 
+  .combat-stats-popup {
+    .suggestion-prompt();
+    .add-tag > div {
+      flex-flow: row;
+      align-items: center;
+      > * {
+        margin: @extra-small-spacer;
+      }
+    }
```
Comment (cynicaloptimist) [inline lesscss/player-view.less]: I think this can be deleted
